src/App.jsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
```

Comment:
I think it would be better to rewrite this part according to the checklist

<Sum a={2} b={3} /> instead of 

<Sum
  a={2}
  b={3}
/>

Reply:
![image](https://github.com/mate-academy/react_sum/assets/123010302/97aafaae-73df-4ea4-a966-4d6efd451a90)